x-pack/plugins/endpoint/public/embeddables/resolver/view/process_event_dot.tsx

```diff
@@ -171,8 +163,8 @@ export const ProcessEventDot = styled(
             viewBox="-15 -15 90 30"
             preserveAspectRatio="xMidYMid meet"
             role="treeitem"
-            {...levelAttribute}
-            {...flowToAttribute}
+            aria-level={adjacentNodeMap.level}
```

Comment:
assign these directly, as the `adjacentNodeMap` will always be available

Reply:
@oatkiller that's OK for level, but aria-flowto can't be empty/null so I think that one still has to be spread

Reply:
typescript caught this as well. ty